docs/platform/longview/what-is-the-linode-longview-app-for-nginx/index.md

```diff
@@ -0,0 +1,255 @@
+[ ok ] Starting Longview Agent: longview.
+update-rc.d: using dependency based boot sequencing
+        {{</ output >}}
+
+        {{< note >}}
+The automatic configuration sets the status page location to `http://127.0.0.2/nginx_status`.
+        {{< /note >}}
+
+        Refresh Longview in the Cloud Manager to verify that the NGINX tab is now present and collecting data for your Longview client instance.
+
+        If instead you received a failure message similar to the output example, you will need to double-check your NGINX installation, and then do a [manual configuration](#manual-configuration-all-distributions). You can also visit the [Troubleshooting](#troubleshooting) section at the end of this article.
+
+        {{< output >}}
+[FAIL] Reloading web server config: nginx failed!
+        {{</ output >}}
+
+### Manual Configuration (All Distributions)
+
+To enable the NGINX Longview app manually:
+
+1. [SSH into your Linode](/docs/getting-started/#connect-to-your-linode-via-ssh) whose system you are monitoring with Longview.
+
+1.  Add the following lines to your NGINX configuration to enable the status module and set the location of the status page. The lines should be placed within your main configuration file's `http` block (`/etc/nginx/nginx.conf`) or in a separate [site configuration file](/docs/web-servers/nginx/slightly-more-advanced-configurations-for-nginx/#host-multiple-websites).
+
+    {{< file "/etc/nginx/nginx.conf" >}}
+server {
+    listen 127.0.0.1:80;
+    server_name 127.0.0.1;
+    location /nginx_status {
+        stub_status on;
+        allow 127.0.0.1;
+        deny all;
+    }
+}
+
+{{< /file >}}
+
+
+2.  Restart NGINX:
+
+        sudo systemctl restart nginx
+
+3.  Edit `/etc/linode/longview.d/Nginx.conf` to look like the following:
+
+    {{< file "/etc/linode/longview.d/Nginx.conf" >}}
+location http://127.0.0.1/nginx_status
+
+{{< /file >}}
+
+
+4.  Restart Longview:
+
+        sudo systemctl restart longview
+
+5.  Refresh Longview in the Cloud Manager to verify that the NGINX tab is now present and collecting data for your Longview client instance.
+
+You should now be able to see Longview data for NGINX. If that's not the case, proceed to the [Troubleshooting](#troubleshooting) section at the end of this article.
+
+## Viewing Statistics
+
+To see the output for the Longview Nginx App:
+
+1.  Log in to the [Linode Cloud Manager](https://cloud.linode.com/).
+1.  Click on the **Longview** link in the sidebar.
+1.  Select the Longview client instance whose NGINX data you'd like to view.
+1.  Select the **Nginx** tab.
+
+    ![The Longview Nginx App.](nginx-overview.png)
+
+    You'll see the current version of NGINX listed on the upper left-hand side of the page.
+
+    Mouse over a data point to see the exact numbers for that time. With [Longview Pro](docs/platform/longview/pricing/), you can view older time periods for your data. The next sections cover the Longview Nginx App in detail.
```

Comment:
I think this needs a beginning / to the link.

Reply:
done